### PR TITLE
explain precedences in complex expressions

### PR DESCRIPTION
**Pull request by JacobChang**
Update documents to address https://github.com/ponylang/pony-tutorial/issues/324

### CODE REVIEW

**Comment by JacobChang** [comment]
Just realised that my editor did some auto formatting for markdown file, it looks okay to me, if anything wrong, I will fix it.

File: content/expressions/methods.md [hunk 1]
```diff
@@ -245,4 +245,14 @@ They are presented more in-depth in the [Object Literals section](../expressions
...
-__Can I start my method name with 2 (or more) underscores?__ No. If the first character is an underscore then the second one MUST be a lower case letter.
+**Can I start my method name with 2 (or more) underscores?** No. If the first character is an underscore then the second one MUST be a lower case letter.
+
+## Precedence
+
+We have talked about [precedence of operators](../expressions/ops.html#precedence) before, and in Pony, methods have higher precedence than any operators.
+
+To sum up, in complex expressions,
+
+1. Methods have higher precedence than any operators.
```
Comment (jemc) [inline content/expressions/methods.md]: Maybe we should say "Method calls and field accesses" here?

**Comment by SeanTAllen** [comment]
Yes, please remove the formatting changes as they aren't germane to the change otherwise.

File: content/expressions/methods.md (hunk 1 above)
Comment (JacobChang) [reply content/expressions/methods.md]: Yeah, agree

**Comment by SeanTAllen** [comment]
Thanks @JacobChang. Awesome first contribution to the tutorial.

**Pull request merged**